e2e/tests/functional/plugins/plot/stackedPlot.e2e.spec.js

```diff
@@ -76,4 +83,20 @@ test.describe('Stacked Plot', () => {
         await expect(page.locator('.js-elements-pool__tree >> text=swg b')).toHaveCount(0);
         await expect(page.locator('.js-elements-pool__tree >> text=swg c')).toHaveCount(1);
     });
+
+    test('Selecting a child plot shows it\'s properties in the inspector.', async ({ page }) => {
+
+        await page.goto(stackedPlot.url);
+
+        //Click on the 2nd plot canvas - There will be 2 canvases for each swg, so we click on the 3rd canvas
+        await page.locator('canvas').nth(3).click();
+
+        // Expand the elements pool vertically
+        const propertiesInspector = await page.locator('.js-inspector-views');
+        await expect(propertiesInspector.locator('.js-series-properties >> h2')).toContainText("Plot Series");
```

Comment:
We should confirm this with all elements in the stacked plot, and we should verify it works while in edit mode also

Reply:
Done.